source/board/stm32h743ii.c

```diff
@@ -23,10 +23,12 @@
 #include "target_board.h"
 #include "i2c.h"
 #include "udb_version.h"
+#include "aux_task.h"
 
 static void prerun_board_config(void) {
     I2C_DAP_Initialize();
     read_udb_version();
+    create_aux_task();
```

Comment:
a more descriptive name would be udb_extended_features_task, just an idea

Reply:
how about adding a flag here so that when we contribute back the code, people can turn off the extended features easily.